app/hypercerts/new/form-steps.tsx

```diff
@@ -511,6 +513,19 @@ const FormSteps = ({
 
   return (
     <section className="space-y-5">
+      {/* {form.watch("cardImage") && (
+        <div className="border-2 border-slate-300 rounded-lg p-4 mb-4">
+          <h3 className="text-lg font-semibold mb-2">Card Preview</h3>
+          <div className="relative w-[336px] h-[420px] mx-auto">
+            <Image
+              src={form.watch("cardImage")}
+              alt="Hypercert Card Preview"
+              layout="fill"
+              objectFit="contain"
+            />
+          </div>
+        </div>
+      )} */}
```

Comment:
```suggestion
```

Reply:
this was a card preview so I can see the final snapshot of the card without having to mint it

Reply:
But... that's a useful feature no? Can't that just always be displayed on the last step? No need to add extra work for this, buit if it's already there?